src/numericReplies.cpp

```diff
@@ -464,8 +469,8 @@ void testAllNumericReplies(const std::string &serverStartTime,
   send482ChanOPrivsNeeded(client, kModeChannel);
   send501UmodeUnknownFlag(client);
   send525InvalidKey(client, kModeChannel);
-  send696InvalidModeParam(client, testChannel.getNameWithPrefix(),
-                          "+m", "testParam");
+  send696InvalidModeParam(client, testChannel.getNameWithPrefix(), "+m",
+                          "testParam");
```

Comment:
Can I remove this 696 numeric reply ? 472 is used instead of it.
(696 is not defined in RFC1459, it appears in a later version)